docs/source/conf.py

```diff
@@ -263,7 +281,7 @@ def process_docstring(app, what, name, obj, options, lines):
     # Only look at objects that inherit from Django's base model class
     if inspect.isclass(obj) and issubclass(obj, models.Model):
 
-        # Ignore abstract models
+        # Ignore abstract models
```

Comment:
what happened here?

Reply:
Some how that was a non-breaking space; flake8 picked it up.